### PR TITLE
fix(twig): deprecated since version 2.7 replace {%spaceless%} by {%ap…

### PR DESCRIPTION
**Pull request by DocRoms**
My Symfony console warns me I'm few deprecated stuff 😓 .

This bundle use a deprecated version of  `{%spaceless %}` tag. 

Since version `>=2.7` of twig we must to use the `{% apply %}` tag with the `spaceless` filter. 

See [here](https://twig.symfony.com/doc/2.x/filters/spaceless.html) for more informations ;) 

🎁 Edit the composer description : it's not only a SF2 version ;)

### CODE REVIEW

File: src/Resources/views/Collector/redis.html.twig [hunk 1]
```diff
@@ -16,9 +16,9 @@
 
 {% block menu %}
 <span class="label">
-    <span class="icon">{% spaceless %}
+    <span class="icon">{% apply spaceless %}
```
Comment (Oliboy50) [inline src/Resources/views/Collector/redis.html.twig:19]:
```twig
    <span class="icon"><img ...
```

File: src/Resources/views/Collector/redis.html.twig [hunk 2]
```diff
@@ -16,9 +16,9 @@
...
-    {% endspaceless %}</span>
+    {% endapply %}</span>
```
Comment (Oliboy50) [inline src/Resources/views/Collector/redis.html.twig:21]:
```twig
    Container" /></span>
```

**Review by Oliboy50 — COMMENTED** [review]
👋 we already have this kind of discussion in other bundles

just remove the `spaceless` stuff and manually remove spaces between html tags

(because using `apply` is not backward compatible with old twig versions)

**Review by Oliboy50 — APPROVED** [review]
🧞

**Review by Stax — APPROVED** [review]

**Comment by Oliboy50** [comment]
released in [v5.0.2](https://github.com/M6Web/RedisBundle/releases/tag/v5.0.2)

**Pull request merged**